frontend/package.json

```diff
@@ -33,7 +33,8 @@
     "check-format": "prettier --check \"src/**/*.{js,jsx}\"",
     "format": "prettier --write \"src/**/*.{js,jsx}\"",
     "storybook": "storybook dev -p 6006",
-    "build-storybook": "storybook build"
+    "build-storybook": "storybook build",
+    "chromatic": "npx chromatic --project-token=chpt_70630482feb0e8b --patch-build=Caleb-B1...main"
```

Comment:
I think we probably don't want to commit this as it specifies your branch build - could you revert it and just keep it locally?